go-controller/Makefile

```diff
@@ -11,7 +11,7 @@ PKGS ?=
 GOPATH ?= $(shell go env GOPATH)
 TEST_REPORT_DIR?=$(CURDIR)/_artifacts
 export TEST_REPORT_DIR
-GO_VERSION ?= 1.18.4
```

Comment:
our go.mod is still using 1.18 shouldn't we update that to 1.19 as well?

Reply:
I can... It's in the [1.27 rebase](https://github.com/ovn-org/ovn-kubernetes/pull/3593). I was just trying to get these changes in place so that rebase PR would
be able to build and test properly.

Reply:
right I'd suggest a 1.27 rebase test with 1.19 and see if it works - if it does update go.mod updates to 1.19 in this PR itself? if not then it means we need the 1.20 for some dependency..

Reply:
ok, testing locally in my 1.27 rebase branch was fine with 1.19. It's now updated here to be 1.19 as
well. Once this is in, I'll get back to the rebase PR.

Reply:
@jluhrsen With this Makefile change, I am now getting the following issue when doing a `make test`:

```
vagrant@devmaster ~/dev/ovn-kubernetes.git/go-controller master*
❯ make test
podman run -it --rm --security-opt label=disable --cap-add=NET_ADMIN --cap-add=SYS_ADMIN -v /home/vagrant/dev/ovn-kubernetes.git:/go/src/github.com/ovn-org/ovn-kubernetes -w /go/src/github.com/ovn-org/ovn-kubernetes/go-controller -e COVERALLS= -e GINKGO_FOCUS="" quay.io/giantswarm/golang:1.19.6 sh -c "RACE=1 DOCKER_TEST=1 COVERALLS= PKGS="" hack/test-go.sh focus \"\" "
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
```

Any ideas how to work around that?

Reply:
This works:
```
❯ podman run -it --rm --security-opt label=disable --cap-add=NET_ADMIN --cap-add=SYS_ADMIN -v /home/vagrant/dev/ovn-kubernetes.git:/go/src/github.com/ovn-org/ovn-kubernetes -w /go/src/github.com/ovn-org/ovn-kubernetes/go-controller -e COVERALLS= -e GINKGO_FOCUS="" quay.io/giantswarm/golang:1.18.4 sh -c "RACE=1 DOCKER_TEST=1 COVERALLS= PKGS="" hack/test-go.sh focus \"\" "
...
This works!!!
```

```
vagrant@devmaster ~/dev/ovn-kubernetes.git/go-controller master*
❯ podman run -it --rm --security-opt label=disable --cap-add=NET_ADMIN --cap-add=SYS_ADMIN -v /home/vagrant/dev/ovn-kubernetes.git:/go/src/github.com/ovn-org/ovn-kubernetes -w /go/src/github.com/ovn-org/ovn-kubernetes/go-controller -e COVERALLS= -e GINKGO_FOCUS="" quay.io/giantswarm/golang:1.19.6 sh -c "RACE=1 DOCKER_TEST=1 COVERALLS= PKGS="" hack/test-go.sh focus \"\" "
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
error obtaining VCS status: exit status 128
        Use -buildvcs=false to disable VCS stamping.
```

Reply:
hopefully #3643 fixes this